app/assets/javascripts/editor.js

```diff
@@ -147,3 +148,19 @@ $E = {
     else preview_btn.button('reset');
   }
 }
+
+
+
+function urlTransformer(url) {
+  return "https://publiclab.org/dashboard";
+}
+
+function classIdTransformer(name) {
```

Comment:
Can you please explain what this is supposed to be doing?

Reply:
The IdTransformer basically does the same thing, but for html elements. It transforms the ID attribute to the specified id. In this case, null is being returned meaning that the sanitizer just bans all html input.

Reply:
This post might explain better than me. https://code.google.com/archive/p/google-caja/wikis/JsHtmlSanitizer.wiki